pkg/manager/task_clean.go

```diff
@@ -43,10 +43,6 @@ func (t *CleanTask) shouldClean(path string) bool {
 
 	if fileExists && t.getStashFromPath(path) != nil {
```

Comment:
Imho we can also remove the 
`logger.Debugf("File Found: %s", path)`
as it pollutes the log file 
We can then break the if else to a simple if e.g.
```golang
	if !fileExists || t.getStashFromPath(path) ==  nil {
		logger.Infof("File not found. Cleaning: \"%s\"", path)
		return true
	}